Makefile

```diff
@@ -96,4 +96,7 @@ download: ## Recursively "go mod download" on all directories where go.mod exist
 toolchain: ## Install developer toolchain
 	./hack/toolchain.sh
 
-.PHONY: help presubmit install-kwok uninstall-kwok build apply delete test deflake vulncheck licenses verify download toolchain
\ No newline at end of file
+gen_instance_types:
```

Comment:
I'd love to see this target part of the `presubmit` target so that we make sure that we always re-gen these instance types if anything changes in the gen script.

Reply:
@jonathan-innis I already discussed this with @njtran and he asked me to not include it in the `presubmit` target

Reply:
If we include the gen_instance_types in the `make presubmit` target, we'll end up overwriting anything that's in that file path (which is the path that we use for embedding)